### PR TITLE
Integrate application management endpoint - Closes #4489

### LINKED ISSUE #4489: Integrate application management endpoint
### Description
application management is running by mock data and it's time to adopt with v6 endpoint

### Acceptance Criteria
- fetch application default by useBlockchainApplicationMeta and merge with application from redux
- add/update status of the application list from server
- adopt application response changes to the app
- adopt unit test

### PR DESCRIPTION
**Pull request by Balanced02**
### What was the problem?

This PR resolves #4489 

### How was it solved?

- [x] Integrate Applications Management with api
- [x] Fetch default applications from api on app load
- [x] Re-fetch default application when network is changed
- [x] Select application node when users switch application

### How was it tested?

- [x] Browser
- [x] Updated/Added Unit tests

### CODE REVIEW

File: tests/fixtures/blockchainApplicationsManage.js [hunk 1]
```diff
@@ -1,63 +1,81 @@
...
     lastCertificateHeight: 9000,
     isDefault: true,
   },
   {
-    name: 'Colecti',
+    chainName: 'Colecti',
     chainID: 'mi34vyyd12g2lkf0rza1irws',
     state: 'active',
-    apis: [{ rest: 'https://service.colecti.com', rpc: 'wss://service.colecti.com' }],
+    serviceURLs: [{ rest: 'https://service.colecti.com', rpc: 'wss://service.colecti.com' }],
```
Comment (ikem-legend) [inline tests/fixtures/blockchainApplicationsManage.js]: @Balanced02 Please update the `rpc` property here to `ws` as well as other cases where it's used

File: setup/react/app/index.js [hunk 2]
```diff
@@ -40,8 +47,18 @@ const App = ({ history }) => {
...
+    // Initialize client on first render to get default application
+    client.create({ http: 'http://165.227.246.146:9901' })
   }, []);
 
+  useEffect(() => {
+    if (!isLoading && data) {
+      data.data.map(dat => setApplication(dat))
+      setCurrentApplication(data.data[0])
+    }
+
```
Comment (ikem-legend) [inline setup/react/app/index.js]: @Balanced02 This empty line can be removed

**Review by ikem-legend — DISMISSED** [review]

File: setup/react/app/index.js [hunk 3]
```diff
@@ -32,6 +36,9 @@ const App = ({ history }) => {
   const dispatch = useDispatch();
   const [loaded, setLoaded] = useState(false);
   const theme = useSelector((state) => (state.settings.darkMode ? 'dark' : 'light'));
+  const { data, isLoading } = useBlockchainApplicationMeta();
```
Comment (eniolam1000752) [inline setup/react/app/index.js]:
```suggestion
  const { data: chainMetaData, isLoading } = useBlockchainApplicationMeta();
```

File: setup/react/app/index.js [hunk 4]
```diff
@@ -40,8 +47,17 @@ const App = ({ history }) => {
     dispatch(bookmarksRetrieved());
     dispatch(settingsRetrieved());
     dispatch(watchListRetrieved());
+    // Initialize client on first render to get default application
+    client.create({ http: 'http://165.227.246.146:9901' })
   }, []);
 
+  useEffect(() => {
+    if (!isLoading && data) {
+      data.data.map(dat => setApplication(dat))
```
Comment (eniolam1000752) [inline setup/react/app/index.js]:
```suggestion
      chainMetaData.data.map(data=> setApplication(data))
```

File: setup/react/app/index.js [hunk 5]
```diff
@@ -40,8 +47,17 @@ const App = ({ history }) => {
...
+      setCurrentApplication(data.data[0])
```
Comment (eniolam1000752) [inline setup/react/app/index.js]:
```suggestion
      setCurrentApplication(chainMetaData.data[0])
```

File: src/modules/network/utils/getNetwork.test.js [hunk 6]
```diff
@@ -8,6 +8,9 @@ describe('Utils: getNetwork', () => {
   describe('getNetworksList', () => {
     const response = [
       { label: 'Mainnet', name: 'mainnet' },
+      { label: 'Alphanet', name: 'alphanet' },
+      { label: 'Betanet', name: 'betanet' },
```
Comment (eniolam1000752) [inline src/modules/network/utils/getNetwork.test.js:12]: just a clarification for a chain other then main chain would they always have this network types

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/network/utils/getNetwork.test.js (hunk 6 above)
Comment (Balanced02) [reply src/modules/network/utils/getNetwork.test.js:12]: Yeah, they would. These are from the service team; so it's believed they all would have same network types. Although these seems like they'll change soon.

Till then, this works fine atm.

**Review by eniolam1000752 — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**